ReactCommon/react/renderer/animations/LayoutAnimationKeyFrameManager.cpp

```diff
@@ -1241,7 +1241,7 @@ void LayoutAnimationKeyFrameManager::queueFinalMutationsForCompletedKeyFrame(
           break;
         case ShadowViewMutation::Type::Update:
           mutationsList.push_back(ShadowViewMutation::UpdateMutation(
-              prev, finalMutation.newChildShadowView));
+              prev, finalMutation.newChildShadowView, {}));
```

Comment:
Is `{}` ok as a default here? What if the layout animation uses `needsCustomLayoutForChildren` too?

Reply:
I said in the description, I used `{}` when I wasn't sure where to get the  parent shadow view from. I hoping the React Native team will help me with this

Reply:
I've filled in some more of the parent shadow views. There's now only one missing in the `calculateShadowViewMutations` of `Differentiator`.  I think that might be ok because they're root shadow views so might not have a parent?